spring-cloud-aws-autoconfigure/src/main/java/org/springframework/cloud/aws/autoconfigure/context/ContextCredentialsAutoConfiguration.java

```diff
@@ -32,6 +33,7 @@
  * @author Agim Emruli
  */
 @Configuration
+@ConditionalOnAwsCloudEnvironment
```

Comment:
I don't agree with this one. 
Why should I need EC2 if I just would like just to have access to AWS ?